src/main/java/org/myrobotlab/service/data/Orientation.java

```diff
@@ -11,6 +11,7 @@ public class Orientation {
   public Double roll = null;
   public Double pitch = null;
   public Double yaw = null;
+  public String src = null;

```

Comment:
should have getters/setters for src if we want it as a member here.

Reply:
Its silly to have both setters and getters and public fields ... I'll remove the setters/getters